### PR TITLE
Blacklist transfer manager

### PR DESCRIPTION
**Pull request by victorwiebe**

### CODE REVIEW

File: src/factories/contractFactory.ts [hunk 1]
```diff
@@ -267,18 +268,36 @@ export default class ContractFactory {
...
+        },
     );
     return contract;
   }
 
+
   public async getLockUpTransferManagerContract(address: string): Promise<LockUpTransferManagerContract> {
     assert.isETHAddressHex('address', address);
-    return new LockUpTransferManagerContract(address, this.provider, this.contractDefaults);
+    const contract = new LockUpTransferManagerContract(address, this.provider, this.contractDefaults);
```
Comment (monitz87) [inline src/factories/contractFactory.ts]: Could you do this same change in this file's `getVestingEscrowWalletContract`? We missed it in the last PR

File: src/contract_wrappers/modules/transfer_manager/blacklist_transfer_manager_wrapper.ts [hunk 2]
```diff
@@ -0,0 +1,777 @@
...
+
+interface AddNewInvestorToNewBlacklistParams extends BlacklistTypeParams {
+  investor: string;
+}
+
+interface BlacklistTypeMultiParams extends TxParams {
+  startTimes: Date[];
+  endTimes: Date[];
+  blacklistNames: string[];
+  repeatPeriodTimes: number[];
```
[the remark this answers is not in the document]
Comment (victorwiebe) [reply src/contract_wrappers/modules/transfer_manager/blacklist_transfer_manager_wrapper.ts]: Seems there already is indication repeatPeriodTimes is in days

**Pull request merged**